github-action.py

```diff
@@ -429,11 +440,12 @@ def build(self):
             )
 
     def upload(self):
-        actual_commit_sha = self.components[0].get_source_commit_hash()
+        actual_commit_sha = self.component.get_source_commit_hash()
         if self.commit_sha != actual_commit_sha:
-            raise AutoActionError(
```

Comment:
This is to prevent full stop for all dists?

Reply:
Yes, exactly. When component is matched using git repo name too, then one name can match multiple components (for example linux-kernel and linux-kernel-latest). The commit id will match only to one (or some) of them, those not matching should be skipped instead of preventing matching from being processed.

Reply:
Hmm, in fact in the most recent version, I changed how multiple components are processed - now separate `cli` instances are used. So, if you prefer, I can keep exception here, but catch it in `for cli in cli_list` loop to allow other components being processed. Any preference?

Reply:
Somehow I like raising exception to be sure we can manage it in the same place where we would catch all them.

Reply:
Changed.